Library/Formula/poco.rb

```diff
@@ -12,18 +14,29 @@ class Poco < Formula
   end
 
   option :cxx11
+  option :universal
 
   depends_on "openssl"
+  depends_on "cmake" => :build
 
   def install
     ENV.cxx11 if build.cxx11?
 
-    arch = Hardware.is_64_bit? ? "Darwin64": "Darwin32"
-    system "./configure", "--prefix=#{prefix}",
-                          "--config=#{arch}",
-                          "--omit=Data/MySQL,Data/ODBC",
-                          "--no-samples",
-                          "--no-tests"
-    system "make", "install", "CC=#{ENV.cc}", "CXX=#{ENV.cxx}"
+    args = ["-DENABLE_DATA_MYSQL=OFF", "-DENABLE_DATA_ODBC=OFF"]
+
+    if build.universal?
+      ENV.universal_binary
+      args << "-DCMAKE_OSX_ARCHITECTURES=#{Hardware::CPU.universal_archs.as_cmake_arch_flags}"
+    end
+
+    # Fix Foundation library version
+    inreplace "Foundation/CMakeLists.txt",
+              "VERSION ${PROJECT_VERSION}",
+              "VERSION ${SHARED_LIBRARY_VERSION}" if !build.head?
+
+    mktemp do
+      system "cmake", buildpath, *args, *std_cmake_args
```

Comment:
Rather than `*args, *std_cmake_args` make the `std_cmake_args` part of the args, with something like:

```ruby
args = std_cmake_args
args << "-DENABLE_DATA_MYSQL=OFF" << "-DENABLE_DATA_ODBC=OFF"
```